src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -569,6 +569,15 @@ public string DefaultPythonEngine
             }
         }
 
+        /// <summary>
+        /// Sets the visibility of the preferences menu option inside the Dynamo menu
+        /// The option will be disabled if in debug mode
+        /// </summary>
+        public bool IsVisiblePreferences
```

Comment:
Can we use converter https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/UI/Converters.cs? Any new property here may end up being APIs we do not intend to expose...

Reply:
Sure, will look into it

Reply:
Updated! Thanks for the comment, I didn't know about that converters.cs file before